BackofficeBundle/Resources/public/ecmascript/Lib/jquery.initialize.js

```diff
@@ -37,7 +37,7 @@
         },
 
         'removeSelector': function(selector) {
-            if (this.callbacks.hasOwnProperty(selector) !== 'undefined') {
+            if (this.callbacks.hasOwnProperty(selector)) {
```

Comment:
remove this modification of this pull request and add it in https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2336/files